hadoop-hdds/docs/content/feature/Quota.md

```diff
@@ -73,10 +87,35 @@ This behavior changes the quota for Bucket1 to 10GB
 
 Total bucket quota should not be greater than its Volume quota. If we have a 10MB Volume, The sum of the sizes of all buckets under this volume cannot exceed 10MB, otherwise the bucket set quota fails.
 
+### Namespace quota
+Namespace quota is a number that represents how many unique names can be used. This number cannot be greater than LONG.MAX_VALUE in Java.
+
+#### Volume Namespace quota
+```shell
+bin/ozone sh volume create --namespace-quota 100 /volume1
+```
+This means setting the namespace quota of Volume1 to 100.
+
+```shell
+bin/ozone sh volume setquota --namespace-quota 1000 /volume1
+```
+This behavior changes the namespace quota of Volume1 to 1000.
+
+#### Bucket Namespace quota
+```shell
+bin/ozone sh bucket create --namespace-quota 100 /volume1/bucket1
+```
+That means bucket1 allows us to use 100 of namespace.
+
+```shell
+bin/ozone sh bucket setquota --namespace-quota 1000 /volume1/bucket1 
+```
+This behavior changes the quota for Bucket1 to 1000.
+
 #### Clear the quota for volume and bucket
 ```shell
-bin/ozone sh volume clrquota --space-quota /volume1
-bin/ozone sh bucket clrquota --space-quota /volume1/bucket1
+bin/ozone sh volume clrquota --space-quota --namespace-quota /volume1
+bin/ozone sh bucket clrquota --space-quota --namespace-quota /volume1/bucket1
```

Comment:
**There's something wrong with the directory structure. At present is:**

#space quota
……
#namespace quota
……
## clear quota
## check quota and usage

**We can change it to:**

#space quota
……
## clear quota
## check quota and usage

#namespace quota
……
## clear quota
## check quota and usage

**or**

#space quota and namespace quota

……
## clear quota
## check quota and usage